### PR TITLE
:construction_worker: ci improvements

### PR DESCRIPTION
**Pull request by nothingelsematters**

### CODE REVIEW

**Comment by nothingelsematters** [comment]
Oh shit, here we go again.

File: README.md [hunk 1]
```diff
@@ -1,7 +1,6 @@
 # Pull party bot
 
 [![Travis CI](https://img.shields.io/travis/com/pool-party/pull-party-bot?logo=travis)](https://travis-ci.com/pool-party/pull-party-bot)
-[![codecov](https://codecov.io/gh/pool-party/pull-party-bot/branch/master/graph/badge.svg)](https://codecov.io/gh/pool-party/pull-party-bot)
```
Comment (ShuffleZZZ) [inline README.md:4]: We don't have code coverage anymore? Or is it just a badge

Comment (nothingelsematters) [reply README.md:4]: Time to say goodbye...

**Comment by nothingelsematters** [comment]
> How many times should I review this

U can review it as many times as u wish, but approve it, plz.

**Comment by RevealMind** [comment]
Why do we add only ci features?!

**Comment by nothingelsematters** [comment]
> Why do we add only ci features?!

This isn't only about ci, m8

**Comment by ShuffleZZZ** [comment]
@nothingelsematters you better merge it while you can

**Pull request merged**